### ISSUE TITLE
Update designs for enforcement matters index page

### ISSUE THREAD

**Issue by jenniferthibault** [18F/fec-eregs#242]
The older mockups are out of date with the current patterns enough that it's worth a fresh mockup.
- Old mockup: https://gsa.invisionapp.com/share/AR7U3Y662#/screens/167711270

Splitting this into a separate issue because I'm having trouble following in the implementation issues. When these get 👍 'ed & questions are sorted here, I'll carry over final designs to the implementation issues.
 
- [x] Mockup for desktop
- [ ] Mockup for mobile

**Comment by porta-antiporta** [comment]
@jenniferthibault we can close this story right?

https://gsa.invisionapp.com/share/S58HMFKMB
https://gsa.invisionapp.com/share/MA8ID5U6Z

**Comment by jenniferthibault** [comment]
Hey @porta-antiporta — sorry for being out of sync on this, but no this issue isn't completed yet. I added that mockup but hadn't reviewed/discussed it with anyone yet (because I was trying to focus on the priorities of last sprint being the MUR search & canonical page). So I just moved it back to the "In progress" column.

The design here has a couple structural changes I'd like to look at with @adborden this week before calling this approach the one we're going to go with. 

I'm going to come back to this & detail out what those are, but noticed pings on a couple related issues that I want to address before they get too far ahead.  BRB!

**Comment by jenniferthibault** [comment]
KK, here's what I want to think through with you, @adborden:  

We talked last week about "transitional" patterns for parts of the legal side that don't yet have all the faceted search functionality, but some day will. When I was looking at Ed's first design for the MUR landing page, this is definitely one where the most up-to-date patterns will conflict with the available functionality right now. Here's what I mean:

The search panel & the article page header styles aren't really two templates we've blended before, and their structures aren't well matched. On the left is Ed's original mockup. On the right is that content shimmied into our existing templates:

![screen shot 2016-09-13 at 11 48 06 am](https://cloud.githubusercontent.com/assets/11636908/18480815/0f24d8bc-79a8-11e6-9e42-bc524f4c0ae6.png)

Then, if we used our most updated templates and returned search results after someone applies filters, the whole main content area has to make a pretty jarring layout jump:

![screen shot 2016-09-13 at 11 57 18 am](https://cloud.githubusercontent.com/assets/11636908/18481181/4c0c628a-79a9-11e6-9d1a-2793b4965e33.png)


I think removing the filter panel from the landing pages will do a few beneficial things:
- bring the focus back to the content of the landing page, as opposed to the attention-grabbing knobs and buttons of the filters
- Make clear that there are two distinct pages/ways to navigate: a landing page, which can help you browse and understand the content, and an advanced search, which can help you find a specific thing.
- Makes the whole page change when going from landing page > search, instead of just part of the page
- Make it easier to transition when we are able to add the facets into the search functionality, since the most up-to-date page layouts are already in place
- de-clutter the landing pages a bit

So I was thinking that if we used something closer to an existing article template, we'd have an easier, less custom transition:

![screen shot 2016-09-13 at 11 59 44 am](https://cloud.githubusercontent.com/assets/11636908/18481296/b0c9954e-79a9-11e6-9efa-73537db5a39b.png)




We could also use this approach on the AO landing page, and AO search results page structures, depending on what you think, @adborden .

**Comment by adborden** [comment]
Yes, awesome! Very much agree to follow the article-style template for the landing pages. I would also like to keep the other landing pages consistent, so also adopt the article template for AOs and Statutes (and regulations at some point?).

The only comment/question I have, is the sidebar search a pattern we're using? I like it, but I wonder if search should be more prominent? It's not the primary action of the page, so I'm not thinking like hero-size search, but wondering if it should fall within the main content.

On other pages, we put search in the header bar where it's not super prominent, but it catches your attention. Maybe I'm wrong on this, but the sidebar usually feels like related/supplemental content and resources... but search doesn't really fit that pattern to me because you're not searching related resources, you're searching the _current_ resources, enforcement matters.

I don't feel strongly on this, but at least want to provoke your brilliant thinking on it :)

**Comment by jenniferthibault** [comment]
@adborden I'm about to run into a meeting, but wanted to get you a quick-ish answer. It is an existing pattern, but now quite in the same way. 

We have a search in the side/resources panel on the Reg/Reporting essentials pages so that people can find existing committees. https://beta.fec.gov/registration-and-reporting/essentials-house-and-senate-candidates-and-committees/

HOWEVER. Smart point that there is possibly a different, better way to integrate search, and I want to think on that a minute. I think you're right, it could be smarter. I'll follow up here with what comes of that.

**Comment by jenniferthibault** [comment]
Ok! Seriously thanks for the prompt @adborden what I'm about to show you feels like so much better work. 

I brought in @emileighoutlaw 's draft of real content for the Enforcement landing page from #233 , and that helped put the design in better context too. 

### The medium-term designs
-  These designs for the landing pages place the search bar (that searches only within that section) more in-line with the content. Users could enter a search term, or click "advanced search" to be taken to the faceted search results page, where they could apply all the fancy things (when they are available).
- The base pattern for this search layout has already been set up on the data side here: https://beta.fec.gov/data/

_(Click to make the image bigger!)_
![screen shot 2016-09-15 at 11 50 21 am](https://cloud.githubusercontent.com/assets/11636908/18556843/ad5e0c90-7b3a-11e6-884f-cda9b03c63ee.png)
 
### The short-term designs
- Since the MURs are the only searchable section within enforcement right now, I think we need to move the search to be more clearly under just MURs.
- Since facets/filters aren't yet available on any data set, we should also remove the "Advanced" search link, until we can offer that functionality. The search results would appear in the same page template mentioned above, but we wouldn't be giving the appearance of more advanced search until it's really available.

_(Click to make the image bigger!)_
![screen shot 2016-09-15 at 11 56 39 am](https://cloud.githubusercontent.com/assets/11636908/18557193/fb26cec0-7b3b-11e6-9b54-6ac42ccc31f3.png)

What do you think?

I know these screenshots are low-res and would be hard to actually build from. I mostly wanted to show structure and layout here to get alignment between us. Once we're there, I'll export them to InVision so that you can see the details better.

**Comment by adborden** [comment]
:+1: Great, this direction is fantastic!

> Since the MURs are the only searchable section within enforcement right now, I think we need to move the search to be more clearly under just MURs.

Yeah, completely agree.

One comment, I like the full-width search pattern on the medium-term Enforcement page. When you linked to the https://beta.fec.gov/data/ page, I _think_ you were referring to the full-width search pattern on that page, but also want to point out that the feedback components are doing something similar that we're going for with search (my opinion at least) in terms of the content flow. The feedback elements are container width rather than the text-column width like you have in your other low-res mocks. I don't know if that was intentional or not, but I think that considering a container width search pattern might also work.

**Comment by jenniferthibault** [comment]
Ah yes, sorry for ambiguous-linking, and thank you for checking!

I was indeed looking at the full-width search pattern as the starting point for the eventual Enforcement page. The reason we might eventually want to use this full-width element on that page, but container-width elements on the other landing pages mostly has to to with how much (and what type of) content is on the rest of each page. Enforcement has more content with different layouts, and we still need the search element to feel like it applies to allllll the stuff. 

SO... yes! Can we use container-width elements for now, then later consider a full page-width element for the Enforcement page (contingent on if/how the content changes, of course)?

I'll get the InVision updated now too so you can see details better.

**Comment by jenniferthibault** [comment]
InVision updated:

[Enforcement landing](https://gsa.invisionapp.com/share/ZX8MON7AH)

[Advisory opinions landing](https://gsa.invisionapp.com/share/268MONKUT)

[Statutes landing](https://gsa.invisionapp.com/share/ND8MONUF4)

**Steps to completion**
- @adborden any adjustments to the approach?
- @emileighoutlaw any adjustments to the layout of the content on the Enforcement page?

**Comment by adborden** [comment]
:+1: awesome, makes sense and looks great. Thanks!

**Comment by emileighoutlaw** [comment]
I think that layout looks great. I love the visual distinction between the content and the searchable doc types.

**Comment by porta-antiporta** [comment]
:tada: @adborden couple questions:
1. IIRC, the asset types that reside on the open FEC web app each have their own template for the asset landing page.  Were you planning on updating all those templates or do you need a separate story for each asset?
2. For regulations, where the landing page <domain>/regulations is actually part of the eregs app; what's the lift to mimic a similar approach?

**Comment by adborden** [comment]
> 1. IIRC, the asset types that reside on the open FEC web app each have their own template for the asset landing page. Were you planning on updating all those templates or do you need a separate story for each asset?

Yeah, they each have their own template. I'll make all the changes as part of the Enforcement matters landing story, it's not hard to update the others. No  new stories necessary.

> 2. For regulations, where the landing page /regulations is actually part of the eregs app; what's the lift to mimic a similar approach?

This would be a small to moderate lift, but I would love to do it, pending a design and copy for it. That page is sadly very inconsistent with the others :(

**Comment by jenniferthibault** [comment]
I have also been leaving all design tasks for the Regulations viewer aside because I understood it to be too hard to change at the moment. 

On the immediate issue, the enforcement landing page designs are complete, so I'm closing this issue and updating https://github.com/18F/fec-eregs/issues/224 with the 👍
